packages/pypy3

```diff
@@ -0,0 +1,66 @@
+require 'package'
+
+class Pypy3 < Package
+  description 'pypy is fast, drop-in replacement of cpython that can execute code over 4 times faster.'
+  homepage 'https://www.pypy.org/'
+  version '7.3.3'
+  compatibility 'x86_64, x86, aarch64'
```

Comment:
I'm not sure if we do a trim but it's probably better if we don't have spaces.  Was this tested?

Reply:
This was tested. I switched text editors and i'm just noticing the mixtures of tabs and spaces. i'm fixing that right now.

Reply:
In any event, `x86` should be `i686` instead and `armv7l` should be added if it works on `aarch64`.  I'm guessing you could replace with `compatibility 'all'`?